### PR TITLE
infamousPlugins: init at 0.2.03

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/infamousPlugins/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchFromGitHub {
+    owner = "ssj71";
+    repo = "infamousPlugins";
+    rev = "v${version}";
+    sha256 = "1pb7vmc703j25rnyx81cqlfzi66v7gm4a49s06dbgy8a66s1i2zl";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ cairomm cmake lv2 libpthreadstubs libXdmcp libXft ntk pcre fftwFloat zita-resampler ];
```
Comment (Infinisil) [inline pkgs/applications/audio/infamousPlugins/default.nix]: Shouldn't cmake be in nativeBuildInputs too?

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -2846,6 +2846,8 @@ with pkgs;
 
   inform7 = callPackage ../development/compilers/inform7 { };
 
+infamousPlugins = callPackage ../applications/audio/infamousPlugins { };
```
Comment (Infinisil) [inline pkgs/top-level/all-packages.nix]: indentation off

File: pkgs/applications/audio/infamousPlugins/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  meta = with stdenv.lib; {
+    homepage = http://ssj71.github.io/infamousPlugins;
```
Comment (Infinisil) [inline pkgs/applications/audio/infamousPlugins/default.nix]: s/http/https

**Review by infinisil — CHANGES_REQUESTED** [review]
Builds fine, but I have no idea what this can be used for, would be nice to have some more info on that.

**Comment by magnetophon** [comment]
@Infinisil Thanks for the feedback, all done.

**Review by infinisil — APPROVED** [review]

**Pull request merged**